app/frontend/src/routes.ts

```diff
@@ -127,3 +127,9 @@ export const routeToCommunity = (
   slug: string,
   page?: CommunityTab
 ) => `${communityBaseRoute}/${id}/${slug}${page ? `/${page}` : ""}`;
+export const editCommunityPageRoute = `${communityRoute}/edit`;
```

Comment:
With this setup, when editing the page, the "Overview" tab is selected. Would be better if local info was.

Reply:
That's not the case if you look at the screenshot, because `routeToEditCommunityPage` sets the `page` param in the route to "info", so if you click from the link, it works fine. However it doesn't stop people from going `/community/1/world/some-gibberish/edit` and still hitting the edit info page form. Hmm maybe rendering the "not found page" in those cases instead is best